documentation/breaking-changes/breaking-changes-attribute-help.md

```diff
@@ -0,0 +1,488 @@
+# Breaking Changes Attribute Help
+
+Below is description of the various types of Breaking Change Attributes (custom attributes) that can be used to decorate the cmdlet or its paramters to call out various [types of breaking changes](https://github.com/praries880/azure-powershell/blob/breakingchangeattribute/documentation/breaking-changes/breaking-changes-definition.md).
+
+## The different types of attributes
+
+- [Common stuff and The generic breaking change attribute](#common-stuff)
+- [Cmdlet deprecation attribute](#cmdletdeprecationattribute)
+- [Cmdlet output breaking change attribute](#cmdletoutputbreakingchangeattribute)
+- [Cmdlet parameter properties change attribute](#cmdletparameterbreakingchangeattribute)
+
+## Common stuff
```

Comment:
@praries880 I would update this header (and any reference in hyperlinks) to use something instead of "stuff" ☺️ , something like "Common Attribute" or "Common Information"

Reply:
done